packages/jest-snapshot/src/plugins.js

```diff
@@ -13,13 +13,15 @@ import prettyFormat from 'pretty-format';
 import jestMockSerializer from './mock_serializer';
 
 const {
+  DOMCollection,
   DOMElement,
   Immutable,
   ReactElement,
   ReactTestComponent,
 } = prettyFormat.plugins;
 
 let PLUGINS = [
+  DOMCollection,
```

Comment:
Although the destructuring assignment above is in alphabetic order, in the array let’s move `DOMCollection` to follow `DOMElement` so that the order of testing plugins is descending in expected frequency.